sources/platform/actors/publishing/monetize.mdx

```diff
@@ -138,6 +122,42 @@ You can also choose not to use it, but then you must handle API integration and
 
 :::
 
+### How to attract larger customers of PPE and PPR Actors
+Each user running your Actor belongs to a specific Apify Store discount tier, namely `FREE`, `BRONZE`, `SILVER`, or `GOLD` tier.
+
+When setting pricing for a PPE/PPR Actor, you can define different prices for different tiers. While optional, we strongly recommend offering progressively lower prices for higher discount tiers to attract large enterprise customers who may spend thousands or tens of thousands of dollars on your Actor.
+The unit platform usage costs for higher tiers will also be lower, helping you maintain healthy profit margins, as explained in the [Computing your costs for PPE and PPR Actors](#computing-your-costs-for-ppe-and-ppr-actors) section.
+
+During an Actor run, you can access the user's discount tier through run environment variables or by querying user data via our API. This allows you to potentially provide premium services to users in higher discount tiers.
+
+Apart from these standard tiers listed above, there are further tiers reserved for enterprise customers, such as `PLATINUM` and `DIAMOND`. If you are interested in providing enterprise level services and attracting large customers, please reach out to us.
+
+### Computing your costs for PPE and PPR Actors
+For both PPE and PPR Actors, profit is computed using the formula `(0.8 * revenue) - costs`. In this section, we'll explain how the `costs` component is calculated.
+
+When paying users run your Actor, it generates platform usage in the form of compute units, data traffic, API operations etc. This usage determines the `costs` in the profit formula above. Note that usage by free users is covered by Apify and doesn't affect your costs. To calculate costs in dollars, multiply the unit cost of each service by the quantity consumed. For example, if a run uses 10 compute units (CUs) at $0.5/CU, your cost would be $5.
+
+As mentioned in the [How to attract larger customers of PPE and PPR Actors](#how-to-attract-larger-customers-of-ppe-and-ppr-actors) section, the user's discount tier determines the unit costs applied to their runs if the Actor has tiered discounts set up. Higher tiers have lower unit costs, allowing you to offer lower prices to high-tier customers while maintaining healthy profit margins.
+
+The table below summarizes the unit prices for cost computation across different discount tiers.
+
+| Service                         | Price is per | `FREE`  | `BRONZE` | `SILVER` | `GOLD`  | `PLATINUM` |
```

Comment:
I thought we agreed that we won't show the platinum pricing. Did I understand it wrong? Or maybe this was done before we talked?

Reply:
Yeah, this was done before, but I just haven't had time to update it yet. My apologies, will re-request review once updated.